hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/TokenServiceImpl.java

```diff
@@ -77,13 +77,14 @@ private Schema tokenSchema() {
             @Override
             public Set<StateDefinition> statesToCreate() {
                 return Set.of(
-                        tokensDef(),
-                        onDiskAccountsDef(),
-                        onDiskAliasesDef(),
-                        onDiskNftsDef(),
-                        onDiskTokenRelsDef(),
-                        payerRecordsDef(),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏In the new version it looks like we don't need the `payerRecordsDef()` equivalent and it's removed, am I understanding this right?

Reply:
yes, records are maintained by Jasper's code, so removed it.